### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2576

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: public/media/css/page.css [hunk 1]
```diff
@@ -36,8 +36,8 @@ Global styles and classes
     line-height: 23px;
   }
 
-  video {
-    width: 100%;
+.hero-unit iframe {
```
Comment (mjschranz) [inline public/media/css/page.css]: Spacing.

File: public/index.html [hunk 2]
```diff
@@ -40,13 +40,10 @@
               </p>
               <p class="intro-bar">
                 <a class="button-blue start" href="/templates/basic">Start from scratch</a>
-			         </p>
+               </p>
```
Comment (mjschranz) [inline public/index.html]: space.

File: make.js [hunk 3]
```diff
@@ -378,11 +378,6 @@ function buildCSS(compress) {
     compress: compress
   });
 
-  lessToCSS({
```
Comment (jbuck) [inline make.js:381]: Nit: you can remove the .gitignore line for css/webfonts.css now too

File: public/index.html [hunk 4]
```diff
@@ -40,13 +39,10 @@
...
-			         </p>
+              </p>
             </div>
-            <div class="span5">
-              <video controls preload="none" poster="media/img/poster.png">
-                <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.webm" type="video/webm">
-                <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.mp4" type="video/mp4">
-              </video>
+            <div class="span7">
+              <iframe class="iframe-template" src="http://popcorn.webmadecontent.org/15_" width="560" height="358"mozallowfullscreen webkitallowfullscreen allowfullscreen></iframe>
```
Comment (jbuck) [inline public/index.html]: You need a space between `height="358"mozallowfullscreen`

File: public/index.html [hunk 5]
```diff
@@ -93,12 +87,10 @@
                   <p>Tell the story of a minifig hero by changing the location of Google Maps, text and images.  A great way to learn the concepts of "Web Native Cinema!"</p>
                 </div>
               </div>
-            </div>
-            <div class="row project-items">
               <div class="span5">
                 <h4><a href="/templates/basic/?savedDataUrl=tour.json">Neighbourhood Tour</a></h4>
                 <div class="thumbnail preview-thumbnail">
-                  <img src="thumbnail-tour.png" alt="http://dev.butyr.org/v/gh_.html">
+                  <img src="thumbnail-tour.png" data-butter-preview="http://popcorn.webmadecontent.org/1n_">
```
Comment (jbuck) [inline public/index.html]: Need an alt attribute here

File: templates/basic/lego.json [hunk 6]
```diff
@@ -1,341 +1,381 @@
...
+      "name": "video-container",
+      "element": "video-container"
+    }
+  ],
+  "media": [
+    {
+      "id": "Media0",
+      "name": "Media0",
+      "url": [
+        "http://videos-origin.mozilla.org/serv/webmademovies/legorough02.webm?butteruid=1350595343539",
```
Comment (jbuck) [inline templates/basic/lego.json]: Use protocol-less or https-only URLs
Also, use the actual CDN server, `http://videos-cdn.mozilla.net/serv/webmademovies/legorough02.webm`
No need for the ?butteruid either.

File: templates/basic/saved-data.json [hunk 7]
```diff
@@ -11,48 +11,92 @@
       "id": "Media0",
       "name": "Media0",
       "url": [
-        "http://videos-cdn.mozilla.net/serv/webmademovies/makecorn.mp4",
-        "http://videos-cdn.mozilla.net/serv/webmademovies/makecorn.ogv",
-        "http://videos-cdn.mozilla.net/serv/webmademovies/makecorn.webm"
+        "http://videos-cdn.mozilla.net/serv/webmademovies/getstarted.mp4?butteruid=1350596729008",
```
Comment (jbuck) [inline templates/basic/saved-data.json]: https and no butteruid

File: templates/basic/soundscape.json [hunk 8]
```diff
@@ -1,276 +1,301 @@
...
+      "name": "video-container",
+      "element": "video-container"
+    }
+  ],
+  "media": [
+    {
+      "id": "Media0",
+      "name": "Media0",
+      "url": [
+        "http://soundcloud.com/brettgaylor/chinatown-tour?butteruid=1350595219950"
```
Comment (jbuck) [inline templates/basic/soundscape.json]: no butteruid

File: templates/basic/tour.json [hunk 9]
```diff
@@ -1,186 +1,212 @@
...
+      "name": "video-container",
+      "element": "video-container"
+    }
+  ],
+  "media": [
+    {
+      "id": "Media0",
+      "name": "Media0",
+      "url": [
+        "http://videos-origin.mozilla.org/serv/webmademovies/neighbourhood2.webm?butteruid=1350595016158",
```
Comment (jbuck) [inline templates/basic/tour.json]: videos-cdn.mozilla.net, https, no butteruid

**Comment by jbuck** [comment]
I can't comment on ted.json because it's too large, but you need to remove the butteruid, and projectID from the json

**Pull request merged**